cli/tests/e2e/targets/extract/js_html_concat.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <script>
+        // ruleid: get-elem-id
+        function myFunction() { document.getElementById("demo").innerHTML = "Paragraph changed."; }
+        var x = foo();
+        // ruleid: get-elem-id
+        function myFunction() { document.getElementById("demo").innerHTML = "Paragraph changed."; }
+        // ruleid: cross-tag-taint
+        bar(x);
+    </script>
+
+    <body>
+        <h2>Demo JavaScript in Body</h2>
+
+        <p id="demo">A Paragraph</p>
+
+        <button type="button" onclick="myFunction()">Try it</button>
+
+        <script>
+            // ruleid: get-elem-id
+            function myFunction() { document.getElementById("demo").innerHTML = "Paragraph changed."; }
```

Comment:
What would happen if we had a `bar(x);` here, would it be a finding of rule `cross-tag-taint` ?